sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Changed type `RecognizePiiEntitiesOptions.DomainFilter` from `PiiEntityDomainType?` to `PiiEntityDomainType`.
 - Changed type `AnalyzeActionsOptions.IncludeStatistics` from `bool` to `bool?`.
 - Removed property `Statistics` from `AnalyzeActionsResult` as it is not currently returned by the service even if the user passes `IncludeStatistics  = true`.
+- Removed property `StringIndexType` from `TextAnalyticsRequestOptions`.This SDK will keep using `UTF-16` code unit as the default encoding.
```

Comment:
nit: missing blank space before "This SDK..."

Reply:
Great eyes!